user_docs/en/userGuide.md

```diff
@@ -3806,19 +3806,23 @@ Follow the directions in [Creating a portable copy](#CreatingAPortableCopy) for
 ### Run COM registration fixing tool... {#RunCOMRegistrationFixingTool}
 
 Installing and uninstalling programs on a computer can, in certain cases, cause COM DLL files to get unregistered.
-As COM Interfaces such as IAccessible depend on correct COM DLL registrations, issues can appear in case the correct registration is missing.
+Because accessibility interfaces such as IAccessible depend on correct COM DLL registrations, irregular behavior can appear if the correct registrations are missing.
```

Comment:
```suggestion
Because accessibility interfaces such as IAccessible depend on correct COM DLL registrations, abnormal behavior can appear if the correct registrations are missing.
```